### PR TITLE
Chore: make a bunch of components standalone

### PR DESCRIPTION
**Pull request by jahow**
### Description

The impacted components are:
 * organization-details.component.ts
 * organization-header.component.ts
 * header-record.component.ts
 * record-apis.component.ts
 * record-downloads.component.ts
 * record-metadata.component.ts
 * record-otherlinks.component.ts
 * record-page.component.ts
 * record-related-records.component.ts
 * record-user-feedbacks.component.ts
 * records-list.component.ts
 * organisations.component.ts
 * chart-view.component.ts
 * figure-container.component.ts
 * geo-table-view.component.ts
 * table-view.component.ts
 * layers-panel.component.ts
 * data-view-permalink.component.ts
 * data-view-share.component.ts
 * view-web-component.component.ts
 * data-view.component.ts
 * external-viewer-button.component.ts
 * map-view.component.ts
 * record-meta.component.ts
 * favorite-star.component.ts
 * catalog-title.component.ts
 * language-switcher.component.ts
 * organisation-preview.component.ts
 * organisations-result.component.ts
 * card/api-card.component.ts
 * ghost/content-ghost.component.ts
 * download-item.component.ts
 * downloads-list.component.ts
 * error.component.ts
 * image-overlay-preview.component.ts
 * metadata-catalog.component.ts
 * metadata-contact.component.ts
 * metadata-info.component.ts
 * metadata-quality-item.component.ts
 * metadata-quality.component.ts
 * pagination-buttons.component.ts
 * pagination.component.ts
 * record-api-form.component.ts
 * related-record-card.component.ts
 * user-feedback-item.component.ts
 * copy-text-button.component.ts
 * navigation-button.component.ts
 * star-toggle.component.ts
 * expandable-panel-button.component.ts
 * expandable-panel.component.ts
 * loading-mask.component.ts
 * progress-bar.component.ts
 * spinning-loader.component.ts

### Quality Assurance Checklist

- [ ] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [ ] If **new logic** ⚙️ is introduced: unit tests were added
- [ ] If **new user stories** 🤏 are introduced: E2E tests were added
- [ ] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

### CODE REVIEW

**Comment by jahow** [comment]
Note: I reverted https://github.com/geonetwork/geonetwork-ui/pull/1042 to make the tests pass but this will have to be done separately (if we really do it)

File: libs/ui/widgets/src/lib/ui-widgets.module.ts [hunk 1]
```diff
@@ -3,23 +3,14 @@ import { UtilSharedModule } from '@geonetwork-ui/util/shared'
...
 
 @NgModule({
-  declarations: [
-    ColorScaleComponent,
-    ProgressBarComponent,
-    StepBarComponent,
-    LoadingMaskComponent,
-    SpinningLoaderComponent,
-  ],
+  declarations: [ColorScaleComponent, StepBarComponent],
```
[the remark this answers is not in the document]
Comment (jahow) [reply libs/ui/widgets/src/lib/ui-widgets.module.ts:13]: Good question, my approach was to pull on the thread and see what comes. So I'd start with a component that's not standalone and make it standalone and then see what components are affected and make them standalone in turn, etc. :)

**Comment by jahow** [comment]
thank you @tkohr , indeed many storybook entries were failing! I went through all of them and fixed them.

**Review by tkohr — APPROVED** [review]
Thanks, working fine now :star_struck:

**Pull request merged**